spec/service/form_list_service_spec.rb

```diff
@@ -61,6 +73,28 @@
           ])
         end
       end
+
+      context "when user is super admin" do
```

Comment:
Should we have a test to check the new column is not available for other roles?

Reply:
Which other roles? 
By default it should have two as tested here https://github.com/alphagov/forms-admin/pull/894/files#diff-c5ae5556f8c48f48fff536926172d25d520905e4ac7de3bb3681586b3ecaa482R64

this "super admin" restriction is only a short term solution we will remove it once we have tested this in prod to see if there are any performance impact

Reply:
I was thinking of trial and editor, but of course neither of those can do anything with users anyway (`can_manage_user?` in https://github.com/alphagov/forms-admin/blob/main/app/policies/user_policy.rb#L25), so it probably doesn't matter...

Reply:
Actually, when we come to remove the `super_admin` only restriction it might be worth hiding the extra column for `trial_users` because in theory the forms would be created by trial user so a bit pointless having lookups to return the trial users name

Reply:
My thought was if we're testing that column is visible for super-admins, should we be testing that it is not visible for trial and editor users? (sorry I got confused between the form and users list 😅 )

Reply:
We are already testing a "default" number of columns here https://github.com/alphagov/forms-admin/pull/894/files#diff-c5ae5556f8c48f48fff536926172d25d520905e4ac7de3bb3681586b3ecaa482R60-R75

My thought is that when we come to removing the super-admin conditions in the service, these tests would fail and would have to be updating to reflect the new default behaviour. In  this PR, we are only testing the exception of a user being a super-admin all other types should only have 2 columns which is the default.

Reply:
Gotcha, sorry I hadn't parsed that test properly. In that case, LGTM!